source/class/qx/tool/cli/commands/Deploy.js

```diff
@@ -72,10 +72,10 @@ qx.Class.define("qx.tool.cli.commands.Deploy", {
      * fired after deploying. With this event 
      * application can do aditional copying 
      * 
```

Comment:
It‘s a bit unclear what this documents- I assume the properties of an object in the event data ? Should be spelled out more clearly.

Reply:
Yes it's the object in the event data. Any idea how to make it clearer?

Reply:
Just add that „the event data is an object with the following properties:“